src/common/blob/util.js

```diff
@@ -72,20 +72,21 @@ define(['q', './BlobMetadata'], function (Q, BlobMetadata) {
      * @param callback
      * @returns {*}
      */
-    function addAssetsFromExportedProject(logger, blobClient, mainMetadata, callback) {
+    function addAssetsFromExportedProject(logger, blobClient, mainMetadata) {
```

Comment:
A general rule of thumb is that public functions should take a callback when they return a callback